selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -681,6 +681,13 @@ public void waitContextMenu() {
         By.id(ContextMenuFirstLevelItems.NEW.get()), WIDGET_TIMEOUT_SEC);
   }
 
+  /** Waits on context menu item is not visible */
+  public void waitItemOnContexMenuIsNotVisible(ContextMenuItems item) {
```

Comment:
Did you mean `waitContexMenuItemIsNotVisible` name?

Reply:
No, i mean the `Open on github` item is not present in the context menu